core/src/main/java/com/radixdlt/api/system/genesis/PreGenesisHealthInfoService.java

```diff
@@ -79,9 +77,4 @@ public PreGenesisHealthInfoService() {}
   public NodeStatus nodeStatus() {
     return NodeStatus.BOOTING_PRE_GENESIS;
   }
-
-  @Override
-  public EventProcessor<ScheduledStatsCollecting> updateStats() {
```

Comment:
can we now also remove the event class (`ScheduledStatsCollecting`)?

Reply:
yup, I removed it, and here is how GH interpreted it:
<img width="867" alt="image" src="https://github.com/radixdlt/babylon-node/assets/119413677/fadb23e8-cd9a-47bd-a899-969b45d2cc54">
 :D